src/components/listItemComponent/ListItemComponent.jsx

```diff
@@ -0,0 +1,34 @@
+import { useSelector } from "react-redux";
+import { selectRedditReturn } from '../reddit/RedditSlice'
+
+const ListItemComponent = (props) => {
+    const redditApiReturn = useSelector(selectRedditReturn);
+    console.log('redditApiReturn', redditApiReturn)
+
+    if(!redditApiReturn.data) {
+        console.log('api return is not true')
+        return <div>reddit is loading...</div>
+    } else 
+        return (
+            <div>
+                {
+                    redditApiReturn.data.children.map((each, index)=> {
+                        return (
+                            <div key={index}>
+                                <div style={{width: '1000px', height: '100px', backgroundColor: 'white', margin: '10px', key: '{generatedKey}'}}>
+                                    <div style={{display: 'flex', color: 'black'}}>
+                                        <img src={each.data.thumbnail} alt={each.data.title} style={{maxHeight: '100px', maxWidth: '100px'}}/>
```

Comment:
for the next ticket, we'll need to figure out a way to have images have a static height and automatically fill the correct with. And padding. But this is good for now

Reply:
normally would you just do the extra work and do that in the first place?  I would have but mvp right?

Reply:
yeah this is good for a first run